FitGoal/FitGoal/UserProfileConfiguration/ViewControllers/WalkthroughPageViewController.swift

```diff
@@ -11,7 +11,15 @@ import UIKit
 class WalkthroughPageViewController: UIPageViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate {
     
     private var pendingIndex = Int()
-    private var currentIndex = Int()
+    private var currentIndex = Int() {
+        didSet {
+            if currentIndex == (walkthroughViewControllers.count - 1) {
+                nextViewControllerButton.isHidden = true
+            } else {
+                nextViewControllerButton.isHidden = false
+            }
+        }
+    }
     
     private let defaultBottomMargin = CGFloat(-30)
```

Comment:
You don't need to change it now, but it's more natural (i.e more common) to initialise the variables with using literals instead of the initialisers. 

```swift
private var currentIndex = 0 // private var currentIndex = Int()

private let defaultBottomMargin: CGFloat = -30 // private let defaultBottomMargin = CGFloat(-30)
```

In both cases the behaviour is the same, it's just that it's more natural to read.